src/main/java/commands/Exit.java

```diff
@@ -0,0 +1,24 @@
+package commands;
+
+import tasks.Task;
+
+import java.util.ArrayList;
+import java.io.FileWriter;
+import java.io.IOException;
+
+
+public class Exit {
+    public static void saveFile(ArrayList<Task> list) {
+        try {
+            FileWriter fw = new FileWriter("/Users/sherlock/ip/src/userData.txt");
+            for (int i = 0; i < list.size(); i++) {
+                Task task = list.get(i);
+                int isDone = task.getIsDone() ? 1 : 0;
+                fw.write(isDone + " " + task.getUserInput() + System.lineSeparator());
+            }
+            fw.close();
+        } catch (IOException e) {
+            System.out.println("Cannot write to file. Something went wrong");
+        }
+    }
+}
```

Comment:
Perhaps you can put this together with Bye, since Exit is not an explicit command